desktop/core/src/desktop/auth/backend.py

```diff
@@ -273,7 +273,14 @@ class AllowFirstUserDjangoBackend(django.contrib.auth.backends.ModelBackend):
   Allows the first user in, but otherwise delegates to Django's
   ModelBackend.
   """
-  def authenticate(self, username=None, email=None, password=None):
+  def authenticate(self, *args, **kwargs):
+    if py3:
```

Comment:
py3 is not defined? Check the headers of other files for how to use `sys` module to check for the Python version